### ISSUE TITLE
Definitely not my luck day

### ISSUE THREAD

**Issue by philclifford** [UbuntuAccomplishments/accomplishments-snap#1]
It fails on my system: 
```
ubuntu-accomplishments 
Gtk-Message: 09:53:27.298: Failed to load module "appmenu-gtk-module"

(accomplishments-viewer:224619): dbind-WARNING **: 09:53:27.393: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-gdI5r9XxQd: No such file or directory
Couldn't open libEGL.so.1: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /var/lib/snapd/lib/gl/libEGL.so.1)

kernel 5.13.0-21-generic

cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=21.10
DISTRIB_CODENAME=impish
DISTRIB_DESCRIPTION="Ubuntu 21.10"

 snap --version
snap    2.53+21.10ubuntu1
snapd   2.53+21.10ubuntu1
series  16
ubuntu  21.10
kernel  5.13.0-21-generic
snap list
Name                       Version                     Rev    Tracking         Publisher           Notes
android-studio             2020.3.1.24                 115    latest/stable    snapcrafters        classic
ascii-image-converter      1.11.0                      32     latest/stable    zoraiz              -
bare                       1.0                         5      latest/stable    canonical✓          base
bitwarden                  1.29.1                      58     latest/stable    bitwarden✓          -
bottom                     nightly                     419    latest/stable    kz6fittycent        -
bpytop                     1.0.67                      249    latest/stable    kz6fittycent        -
bw                         1.19.1                      32     latest/stable    bitwarden✓          -
code                       ccbaa2d2                    82     latest/stable    vscode✓             classic
core                       16-2.52.1                   11993  latest/stable    canonical✓          core
core18                     20211028                    2253   latest/stable    canonical✓          base
core20                     20211115                    1242   latest/stable    canonical✓          base
cura-slicer                4.12.1                      55     latest/stable    glasen              -
discord                    0.0.16                      130    latest/stable    snapcrafters        -
flutter                    0+git.671983f               111    latest/stable    flutter-team✓       classic
freecad                    0.19                        22     latest/stable    vejmarie            -
gitkraken                  8.1.1                       185    latest/stable    gitkraken✓          classic
gnome-3-26-1604            3.26.0.20210629             104    latest/stable    canonical✓          -
gnome-3-28-1804            3.28.0-19-g98f9e67.98f9e67  161    latest/stable    canonical✓          -
gnome-3-34-1804            0+git.3556cb3               72     latest/stable    canonical✓          -
gnome-3-38-2004            0+git.6ba6040               76     latest/stable    canonical✓          -
go                         1.17.3                      8627   latest/stable    mwhudson            classic
gtk-common-themes          0.1-59-g7bca6ae             1519   latest/stable    canonical✓          -
hexchat                    2.14.2                      42     latest/stable    tingping            -
hollywood                  1.21                        3      latest/stable    kirkland            classic
htop                       3.1.1                       3233   latest/stable    maxiberta           -
hugo                       v0.89.4                     11456  latest/stable    hugo-authors        -
jupyter                    1.0.0                       6      latest/stable    projectjupyter✓     -
microk8s                   v1.21.6                     2646   1.21/stable      canonical✓          classic
musescore                  3.6.2                       216    latest/stable    musescore✓          -
node-red                   2.1.2                       766    latest/beta      noderedteam✓        -
obs-studio                 27.1.3                      1276   latest/stable    snapcrafters        -
openscad                   2021.01                     190    latest/stable    torsten-paul        -
prusa-slicer               2.3.3+snap2                 29     latest/stable    ivo-cavalcante      -
qr-code-generator-desktop  v1.0.27+git2.c64e4cb        304    latest/stable    studiolacosanostra  -
qt513                      5.13                        24     latest/stable    keshavnrj           -
shfmt                      3.3.1                       31     latest/stable    bashfulrobot        -
snapd                      2.52.1                      13640  latest/stable    canonical✓          snapd
software-boutique          0+git.f633ffb               54     latest/stable/…  flexiondotorg       classic
svgo                       2.7.0                       11     latest/stable    ralight             -
ubuntu-accomplishments     0.1                         21     latest/stable    diddledani          -
ubuntu-mate-welcome        21.10.0-5b832ea8            646    latest/stable    flexiondotorg       classic
utm-no                     0+git.b9f523f               34     latest/stable    sil                 -
v4l2-rtspserver            0+git.4ff0c0c               517    latest/stable    mpromonet           -
whatsie                    2.0                         48     latest/stable    keshavnrj           -
wormhole                   0.12.0                      349    latest/stable    snapcrafters        -

```
If you need anything more I'll try and oblige after the 7 hour drive.

**Comment by lucyllewy** [comment]
This should be fixed by 7758da2. It's in the edge channel while I gather feedback. You can install the edge channel with:

```bash
# if already installed:
sudo snap refresh --edge ubuntu-accomplishments

# if you have not installed yet, or you uninstalled it:
sudo snap install --edge ubuntu-accomplishments
```

**Comment by lucyllewy** [comment]
it's now in stable, so you shouldn't need to do anything special other than maybe forcing a refresh:

```bash
sudo snap refresh ubuntu-accomplishments
```

**Comment by philclifford** [comment]
Thanks, working for me on stable now. Just need to accomplish something now.
